modules/ossm-config-otel-kiali.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/kiali/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-otel-kiali_{context}"]
+= Configuring {DTProductName} with Kiali
+
+When {KialiProduct} is properly configured with {DTProductName}, you can view distributed traces in the Kiali console. These traces provide insight into the communication between services within the service mesh, helping you understand how requests are flowing through your system and where potential issues might be.
+
+.Prerequisites
+
+* {SMProductName} is installed.
+
+* You configured {DTShortName} with {SMProductName}.
+
+.Procedure
+
+. Setup the Kiali custom resource to access traces from the {TempoName} frontend:
```

Comment:
```suggestion
. Set up the `Kiali` custom resource to access traces from the {TempoName} frontend:
```
Presumably the CR type is `Kiali`, per https://kiali.io/docs/installation/installation-guide/creating-updating-kiali-cr/ ? If so, it should be formatted this way as an API object per the [OCP doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references)

I'd also maybe instead of saying "Set up" here be more specific - does a user need to "create" or "modify" this CR?

Also, consider adding a step to apply the CR once it's created / modified, using `oc apply -f <filename>.yaml` - probably worth checking that with your eng or QE team but I'd assume it's a required step

This step reads a little confusingly, because it says you're setting up the `Kiali` CR but then the example heading is "Example Tempo frontend" - I think adding the API info / metadata in the example would help provide context, if it's relevant, e.g. if this is a `TempoStack` CR that's being modified:

```
apiVersion: tempo.grafana.com/v1alpha1
kind: TempoStack
metadata:
  name: simplest
spec:
# ...
```

or, if it's a `Kiali` CR being created or modified;

```
apiVersion: kiali.io/v1alpha1
kind: Kiali
metadata:
  name: kiali
spec:
# ...
```

or maybe it's something else entirely, some other config type file? Either way, it's not immediately clear to me which file is being modified here and whether there are additional steps required to apply it after it has been modified.

Reply:
its this but modified for 3.0 https://kiali.io/docs/configuration/p8s-jaeger-grafana/grafana/